.stickler.yml

```diff
@@ -0,0 +1,21 @@
+
+# add the linters you want stickler to use for this project
+linters:
+  eslint:
+    
+    # indicate where is the config file for stylelint
+    config: './.eslintrc.json'
+    
+  sasslint:
+    # indicate where is the config file for stylelint
+    config: ./sass-lint.yml
```

Comment:
Also, You did not even have to style lint. but since you did it's best to fix errors.

Reply:
If fixing the config here does not help I'd try my best to help troubleshoot with you. 🙂